submissions/m-ruslan/html-css-popup/index.html

```diff
@@ -0,0 +1,153 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="app.css" />
+    <link rel="icon" type="image/png" href="favicon.png" />
+    <title>Popup</title>
+  </head>
+  <body>
+    <nav>
+      <ul class="nav-container">
+        <li class="nav-el">
+          <a href="#" tabindex="0">Mail</a>
+        </li>
+        <li class="nav-el">
+          <a href="#" tabindex="0">Images</a>
+        </li>
+
+        <li class="nav-el">
+          <input type="checkbox" id="button-1"  tabindex="0">
+          <a href="#" tabindex="-1">
+            <label for="button-1" id="menu-label">
+              <img src="img/apps_new.png" alt="Menu icon">
+            </label>
+          </a>
+
+          <div id="apps-window">
+            <input type="checkbox" id="button-2" tabindex="2">
+            <button id="more" tabindex="-1">
+              <label for="button-2" id="more-label">
+                More
+              </label>
+            </button>
+            
+            <div id="apps-cont">
+              <div id="apps-block1">
+                <a href="#" class="app-cont" tabindex="1">
```

Comment:
In general (99%) you shouldn't add additional `tabindex` attributes to interactive tags (links, buttons or form elements). 
In this case `tabindex` attribute breaks focusing flow

![Oct-26-2020 12-48-51](https://user-images.githubusercontent.com/42999342/97163663-a4eebd80-1789-11eb-9d64-3573cab2990f.gif)

Reply:
Thank you, I've removed tabindexes from my code and it looks that it works fine without them)

Reply:
It's really better!
But one more thing:
![Oct-30-2020 21-36-24](https://user-images.githubusercontent.com/42999342/97749680-25018400-1af8-11eb-990b-2b5c2d06c622.gif)

You can just move label "under: the links wrapper.
<img width="1012" alt="Screenshot 2020-10-30 at 21 38 21" src="https://user-images.githubusercontent.com/42999342/97749916-86c1ee00-1af8-11eb-8715-678df3ef9769.png">

Reply:
Artem, I've moved 'label' as you showed in the screenshot above, but 'More' button still gets focused before the links.
If I move 'input' too, then I can't manage to change '.apps-cont' styles (or haven't found how to do it yet) when 'input' is checked because '.apps-cont' becomes previous sibling of 'input' in the html tree.

Reply:
Hmm... you're right!
In perfect world, I believe, you have to restructured your element in markup and some styles... but our world isn't perfect 🌏 
You've made a lot of changes and I hope you learned something.

I've marked this issue as "*resolved*"